packages/@aws-cdk/aws-eks/lib/kubectl-handler/helm/__init__.py

```diff
@@ -67,6 +68,8 @@ def helm(verb, release, chart = None, repo = None, file = None, namespace = None
             cmnd.extend(['--version', version])
         if not namespace is None:
             cmnd.extend(['--namespace', namespace])
+        if wait:
+            cmnd.append('--wait')
```

Comment:
We might have a problem here if this takes more than the lambda timeout.

Reply:
We could add a `--timeout` as well, or perhaps call it `--wait-max` that accepts an optional integer value instead? If omitted, it means "don't wait", and if it's an integer, wait for that long.

However, it defaults to `false` right now and the `--wait` timeout is 300 seconds by default (and the integration tests take forever to update! 🙂), so I think it might be OK like this.

Reply:
This is fine like this.